client/pdo/client/controller/contract_controller.py

```diff
@@ -281,6 +291,16 @@ def precmd(self, line) :
         if self.echo:
             print(line)
 
+        if line[-1] == '\\' :
```

Comment:
Same as above. A short descriptive comment would be helpful here.

Reply:
Done